src/Model/RecordLocale.php

```diff
@@ -315,4 +318,53 @@ public function IsDraft()
 
         return true;
     }
+
+    /**
+     * Check if a record is published, but modified on draft
+     * useful to indicate that record has changes that need publishing
+     *
+     * @return bool
+     */
+    public function StagesDiffer()
+    {
+        if (!$this->IsPublished(true)) {
+            return false;
+        }
+
+        /** @var DataObject|FluentVersionedExtension $record */
+        $record = $this->getOriginalRecord();
+
+        // Check if item is versioned
+        if (!$record->hasExtension(FluentVersionedExtension::class)) {
+            return false;
+        }
+
+        return $record->stagesDifferInLocale($this->getLocale());
+    }
+
+    /**
+     * Get the locale which is the source of content for this record
+     *
+     * @return Locale|null
+     */
+    public function SourceLocale()
```

Comment:
Is there a way you can name this something without a capital letter at the start? It will be a blocker for PSR-12 compliance in future